applications/substrate-identity-directory.md

```diff
@@ -95,18 +95,24 @@ Mockups:
 | ------------- | ------------- | ------------- |
 | 1. | Implement logic for sending tokens | Implement logic for sending tokens; retrieve balance, parse inputs, display transaction fee, create the transfer transaction. |
 | 2. | Make the web service work offline | Web service can be used in offline mode; the user can specify a local node to which will the service connect. |
- 
 
 ### Milestone 3 Index data and query
+#### Requirements for the milestone:
+* https://github.com/paritytech/substrate-archive/issues/97
+* https://github.com/paritytech/substrate-archive/issues/111
+* https://github.com/paritytech/substrate-archive/issues/132 
```

Comment:
These are different PRs from the one you mentioned in your previous email, could you please explain? And do you care to elaborate on how this is blocking you exactly? Polkascan for example have implemented pretty much this exact functionality https://polkascan.io/kusama/account/EGVQCe73TpFyAZx5uKfE1222XfkT3BSKozjgcqzLBnc5eYo

Reply:
I have added the missing PR to the list. To be able to retrieve and decode extrinsics from older blocks, this PR and issues must be merged and solved. Those three issues on substrate-archive are related to type-decoding.
Also, this is the response we got from the official discord server:
"At what blocks are you not getting any traces? State tracing is only supported on runtime versions v8.27 and above, so if you are on a block height below that runtime version then you will not get any traces. For runtime v0.9.XX a release is being cut to include support for those runtime versions today or tomorrow."
That is still not live in the latest version.
As agreed on the call, let's proceed with the payment for the first two milestones as they are delivered and not connected to the current issue. Thank you for your input about polkascan, we will research milestone 3 on their method for retrieving the missing data.